core/src/banking_stage/consumer.rs

```diff
@@ -853,6 +866,20 @@ mod tests {
         }
     }
 
+    fn store_nonce_account(
```

Comment:
*minor*

Should this be compiled only for the test configuration:

```suggestion
    #[cfg(test)]
    fn store_nonce_account(
```

This file test organization is a bit unusual.
Normally, tests are put in a separate module, so that dependencies that are used only by the tests do not trigger unused warnings.

```rust
#[cfg(test)]
mod tests {
    use test::only:dependencies;

    fn store_nonce_account() {}

    #[test]
    fn test_bank_process_and_record_transactions_in_last_tick() {}
}

Reply:
All of this code is already in the tests module

Reply:
Ah, sorry %)